---
id: 642
title: Functional Phylogenomics Analysis of Bacteria and Archaea using Consistent Genome Annotation with UniFam.
date: 2014-10-09T22:25:46+00:00
author: omicsbio
excerpt: Functional Phylogenomics Analysis of Bacteria and Archaea using Consistent Genome Annotation with UniFam.
layout: post
guid: https://www.omicsbio.org/?p=642
permalink: /2014/10/09/functional-phylogenomics-analysis-bacteria-archaea-using-consistent-genome-annotation-unifam/
basic_page_layout:
  - full
categories:
  - Publications
tags:
  - Genomics
---
<span style="color: #0000ff;">Juanjuan Chai</span>, Kora, G., <span style="color: #0000ff;">Tae-Hyuk Ahn</span>, Hyatt, D. & <span style="color: #0000ff;">Chongle Pan</span>

<span style="color: #ff0000;"><em>BMC Evolutionary Biology</em></span> 14:207. doi: 10.1186/s12862-014-0207-y (2014).Â  [[Link to Article](http://www.biomedcentral.com/1471-2148/14/207)]

<!--more-->

<div class="collapsible-content">
  <p>
    <strong>Abstract:</strong> Background: Phylogenetic studies have provided detailed knowledge on the evolutionary mechanisms of genes and species in Bacteria and Archaea. However, the evolution of cellular functions, represented by metabolic pathways and biological processes, has not been systematically characterized. Many clades in the prokaryotic tree of life have now been covered by sequenced genomes in GenBank. This enables a large-scale functional phylogenomics study of many computationally inferred cellular functions across all sequenced prokaryotes.
  </p>
  
  <p>
    Results: A total of 14,727 GenBank prokaryotic genomes were re-annotated using a new protein family database, UniFam, to obtain consistent functional annotations for accurate comparison. The functional profile of a genome was represented by the biological process Gene Ontology (GO) terms in its annotation. The GO term enrichment analysis differentiated the functional profiles between selected archaeal taxa. 706 prokaryotic metabolic pathways were inferred from these genomes using Pathway Tools and MetaCyc. The consistency between the distribution of metabolic pathways in the genomes and the phylogenetic tree of the genomes was measured using parsimony scores and retention indices. The ancestral functional profiles at the internal nodes of the phylogenetic tree were reconstructed to track the gains and losses of metabolic pathways in evolutionary history.
  </p>
  
  <p>
    Conclusions: Our functional phylogenomics analysis shows divergent functional profiles of taxa and clades. Such function-phylogeny correlation stems from a set of clade-specific cellular functions with low parsimony scores. On the other hand, many cellular functions are sparsely dispersed across many clades with high parsimony scores. These different types of cellular functions have distinct evolutionary patterns reconstructed from the prokaryotic tree.
  </p>
</div>

<p class="gde-text">
  <a href="https://www.omicsbio.org/wp-content/uploads/2014/11/Paper_2014_Chai_UniFam_Phylogenomics.pdf" class="gde-link" onClick="_gaq.push(['_trackEvent', 'Google Doc Embedder', 'Download', this.href]);">Download (PDF, Paper_2014_Chai_UniFam_Phylogenomics.pdf)</a>
</p>