---
layout: default
title: Quantitative metaproteomics: functional insights into microbial communities
date: 2014-02-12T20:53:34+00:00
layout: default
parent: 2014 posts
grand_parent: Posts
---
# Quantitative metaproteomics: functional insights into microbial communities
<span style="color: #0000ff;">Chongle Pan</span> & Banfield, J. F.

<span style="color: #ff0000;"><em>Methods Mol Biol</em></span> **1096**, 231-240, doi:10.1007/978-1-62703-712-9_18 (2014). [[Link to Article](http://link.springer.com/protocol/10.1007%2F978-1-62703-712-9_18)]

<!--more-->

**Abstract:** Quantitative metaproteomics aims to accurately determine the relative abundances of thousands of proteins in a microbial community. This approach can be used to provide a comprehensive view of metabolic activities of organisms in microbial communities and uncover significant changes in protein expression between communities at different developmental stages, environment types or in response to different perturbations. Here, we describe three strategies for quantitative metaproteomics, including label-free, (15)N metabolic labeling, and isobaric chemical labeling. The measurements are all based on a shotgun proteomics workflow involving proteolysis, two-dimensional liquid chromatogram-tandem mass spectrometry, and database searching against a metagenomic protein database. Quantitative metaproteomics was established and demonstrated using a model microbial community from acid mine drainage.