---
id: 986
title: Computational Analysis of Complex Proteogenomics Data for Characterization of Terrestrial Carbon Turnover by Soil Microbial Communities
date: 2016-05-30T22:31:43+00:00
author: omicsbio
layout: post
guid: https://www.omicsbio.org/?p=986
permalink: /2016/05/30/computational-analysis-of-complex-proteogenomics-data-for-characterization-of-terrestrial-carbon-turnover-by-soil-microbial-communities/
categories:
  - Proposals
---
The DOE ASCR ALCC program awarded us 9 million CPU hours on the Titan supercomputer to analyze proteogenomic data from soil communities.

**Summery**:

We developed a range of proteogenomics approaches based on scalable data analytics algorithms and high-performance computing. The Sipros algorithm was developed for metaproteomics to identify proteins and their post-translation modifications and to track the incorporation of labeled substrates into microbial populations. The UniFam protein family database was constructed for high-throughput functional annotation of large metagenomes. The Sigma algorithm was developed for taxonomic classification to large microbial communities. In our previous ALCC project, we have deployed these scalable algorithms on Titan and demonstrated their computational capabilities for enabling biological discoveries.

Here we apply for computational resources on Titan to analyze complex proteogenomics data that are being generated from ongoing studies on complex soil communities in a model grassland ecosystem located in the Angelo reserve in northern California and a tropical forest ecosystem manipulation (17-year nutrient addition) in Panama managed by the Smithsonian Tropical Research Institute. The primary objective of this research is to determine the biological processes used by microbial communities to degrade carbon and the perturbation of these processes by moisture level and nutrient availability. Metagenomics is used to determine the composition of microbial communities and functional types represented by the genes. Metaproteomics is used to compare the protein expression changes in soil samples across different environmental conditions. Proteomic stable isotope probing is used to trace <sup>15</sup>N-, <sup>2</sup>H-, and <sup>13</sup>C-labeled substrates into the microbial community. Integration of these omics results will elucidate an organism-resolved profile of soil carbon cycling by microbial communities and provide new information to enable the incorporation of genomic information into climate models.