---
id: 673
title: Exhaustive database searching for amino acid mutations in proteomes.
date: 2014-11-02T23:06:04+00:00
author: omicsbio
layout: revision
guid: https://www.omicsbio.org/2014/11/02/262-autosave-v1/
permalink: /2014/11/02/262-autosave-v1/
---
Hyatt, D. & <span style="color: #0000ff;">Chongle Pan</span>

<span style="color: #ff0000;"><em>Bioinformatics</em> </span>**28**, 1895-1901, doi:10.1093/bioinformatics/bts274 (2012). [Link to Article]

<!--more-->

**Abstract:** MOTIVATION: Amino acid mutations in proteins can be found by searching tandem mass spectra acquired in shotgun proteomics experiments against protein sequences predicted from genomes. Traditionally, unconstrained searches for amino acid mutations have been accomplished by using a sequence tagging approach that combines de novo sequencing with database searching. However, this approach is limited by the performance of de novo sequencing. RESULTS: The Sipros algorithm v2.0 was developed to perform unconstrained database searching using high-resolution tandem mass spectra by exhaustively enumerating all single non-isobaric mutations for every residue in a protein database. The performance of Sipros for amino acid mutation identification exceeded that of an established sequence tagging algorithm, Inspect, based on benchmarking results from a Rhodopseudomonas palustris proteomics dataset. To demonstrate the viability of the algorithm for meta-proteomics, Sipros was used to identify amino acid mutations in a natural microbial community in acid mine drainage. AVAILABILITY: The Sipros algorithm is freely available at <http://code.google.com/p/sipros>.

<p class="gde-text">
  <a href="https://www.omicsbio.org/wp-content/uploads/2012/07/Paper_2012_Hyatt_Mutation_Search.pdf" class="gde-link" onClick="_gaq.push(['_trackEvent', 'Google Doc Embedder', 'Download', this.href]);">Download (PDF, Paper_2012_Hyatt_Mutation_Search.pdf)</a>
</p>