---
id: 930
title: Phylogenetically conserved resource partitioning in the coastal microbial loop
date: 2017-09-28T17:48:47+00:00
author: omicsbio
layout: revision
guid: https://www.omicsbio.org/2017/09/28/929-revision-v1/
permalink: /2017/09/28/929-revision-v1/
---
S. Bryson, Z. Li, F. Chavez, P. K. Weber, J. Pett-Ridge, R. L. Hettich, C. Pan, X. Mayali, and R. S. Mueller

_The ISME journal_, __ doi: 10.1038/ismej.2017.128. (2017) [[Link to Article](http://dx.doi.org/10.1038/ismej.2017.128)]

Resource availability influences marine microbial community structure, suggesting that population-specific resource partitioning defines discrete niches. Identifying how resources are partitioned among populations, thereby characterizing functional guilds within the communities, remains a challenge for microbial ecologists. We used proteomic stable isotope probing (SIP) and NanoSIMS analysis of phylogenetic microarrays (Chip-SIP) along with 16S rRNA gene amplicon and metagenomic sequencing to characterize the assimilation of six <sup>13</sup>C-labeled common metabolic substrates and changes in the microbial community structure within surface water collected from Monterey Bay, CA. Both sequencing approaches indicated distinct substrate-specific community shifts. However, observed changes in relative abundance for individual populations did not correlate well with directly measured substrate assimilation. The complementary SIP techniques identified assimilation of all six substrates by diverse taxa, but also revealed differential assimilation of substrates into protein and ribonucleotide biomass between taxa. Substrate assimilation trends indicated significantly conserved resource partitioning among populations within the Flavobacteriia, Alphaproteobacteria and Gammaproteobacteria classes, suggesting that functional guilds within marine microbial communities are phylogenetically cohesive. However, populations within these classes exhibited heterogeneity in biosynthetic activity, which distinguished high-activity copiotrophs from low-activity oligotrophs. These results indicate distinct growth responses between populations that is not apparent by genome sequencing alone.