---
id: 701
title: Quantitative proteomic analyses of the response of acidophilic microbial communities to different pH conditions
date: 2014-11-02T23:14:45+00:00
author: omicsbio
layout: revision
guid: https://www.omicsbio.org/2014/11/02/252-revision-v1/
permalink: /2014/11/02/252-revision-v1/
---
Belnap, C. P., <span style="color: #0000ff;">Chongle Pan</span>, Denef, V. J., Samatova, N. F., Hettich, R. L. & Banfield, J. F.

<span style="color: #ff0000;"><em>The ISME journal</em></span> **5**, 1152-1161, doi:10.1038/ismej.2010.200 (2011). [[Link to Article](http://www.nature.com/ismej/journal/v5/n7/full/ismej2010200a.html)]

<!--more-->

**Abstract:** Extensive genomic characterization of multi-species acid mine drainage microbial consortia combined with laboratory cultivation has enabled the application of quantitative proteomic analyses at the community level. In this study, quantitative proteomic comparisons were used to functionally characterize laboratory-cultivated acidophilic communities sustained in pH 1.45 or 0.85 conditions. The distributions of all proteins identified for individual organisms indicated biases for either high or low pH, and suggests pH-specific niche partitioning for low abundance bacteria and archaea. Although the proteome of the dominant bacterium, Leptospirillum group II, was largely unaffected by pH treatments, analysis of functional categories indicated proteins involved in amino acid and nucleotide metabolism, as well as cell membrane/envelope biogenesis were overrepresented at high pH. Comparison of specific protein abundances indicates higher pH conditions favor Leptospirillum group III, whereas low pH conditions promote the growth of certain archaea. Thus, quantitative proteomic comparisons revealed distinct differences in community composition and metabolic function of individual organisms during different pH treatments. Proteomic analysis revealed other aspects of community function. Different numbers of phage proteins were identified across biological replicates, indicating stochastic spatial heterogeneity of phage outbreaks. Additionally, proteomic data were used to identify a previously unknown genotypic variant of Leptospirillum group II, an indication of selection for a specific Leptospirillum group II population in laboratory communities. Our results confirm the importance of pH and related geochemical factors in fine-tuning acidophilic microbial community structure and function at the species and strain level, and demonstrate the broad utility of proteomics in laboratory community studies.