---
id: 1279
title: Research
date: 2019-02-07T19:31:32+00:00
author: omicsbio
layout: revision
guid: https://www.omicsbio.org/2019/02/07/62-revision-v1/
permalink: /2019/02/07/62-revision-v1/
---
## Computer Science Research

#### Analytics of big -omics data with supercomputing

We are interested in developing scalable algorithms for analysis of large-scale sequencing data and mass spectrometry data from genomics, transcriptomics, proteomics and metabolomics. Our approaches are based on graph theory, computational statistics, and machine learning. Our research is focused on parallelization of the data analytics algorithms. We use a hybrid parallelization architecture that combines OpenMP multi-threading, MPI message passing, and MPI remote message access.

#### Machine learning for predictive genomics

We are interested in developing deep learning algorithms to predict complex phenotypes from genomic data. Deep learning models can provide better prediction accuracy than the existing linear models for estimation of polygenic risk scores for complex diseases. However, it is difficult to interpret deep learning models and explain how their accurate predictions are made. Interpretation of deep learning models is critical for domain scientists to verify that the inferences are scientifically sound. Domain scientists can also extract useful knowledge about the data by understanding the signal captured by deep learning models for prediction. We use supercomputing to interpret the prediction of deep learning models.

#### Knowledge discovery from -omics data using network analysis.

We are interested in using a network approach to extract biological knowledge from complex -omics data. A semantic network (knowledge graph) is constructed from existing databases of protein functions, metabolic and regulatory networks, and orthology relationships. Information can be extracting from -omics data by mapping all the measured genes, proteins, and metabolites onto such a semantic network. We develop scalable network analytics algorithms, such as graph pattern matching, community detection, and enrichment analysis, for knowledge discovery.

## Biology Research

#### Characterization of microbial communities using a meta-omics approach

Microbial communities drive many biological processes in diverse ecosystems. We use metagenomics, metaproteomics, and stable isotope probing to study microbial communities ranging from human gastrointestinal tracts to coastal waters and forest soils in collaboration with experts of these ecosystems. The results can provide important insights into the ecology and evolutionary biology of microorganisms in their natural environments. The knowledge on microbial communities can be used to improve the health of human hosts and construct nutrient cycling models of terrestrial and aquatic ecosystems.

#### Inference of metabolic and regulatory networks in eukaryotic organisms.

Fungi, plants, animals, and humans are complex organisms with large genomes and sophisticated regulations of transcriptomes, proteomes and metabolomes. We use integrated -omics analyses to shed light on the metabolic and regulatory networks of these organisms in collaboration with domain experts.  We are also interested in studying the molecular -omics phenotypes of genomic variations stemming from somatic mutations, natural genetic polymorphisms, or genetic engineering. This may improve our understanding of the genetic basis of diseases and traits of plants and humans.