---
id: 515
title: Resources
date: 2018-09-23T12:00:14+00:00
author: omicsbio
layout: revision
guid: https://www.omicsbio.org/2014/09/29/328-autosave-v1/
permalink: /2018/09/23/328-autosave-v1/
---
### Protocols

Please see the [computational and experimental protocols](https://www.omicsbio.org/category/protocols/) used in our research.

### Laboratory space

We have a dry lab within the School of Computer Science in the Devon Energy Hall and a wet lab within the Department of Microbiology and Plant Biology in the George Lynn Cross Hall. The two labs are within 10-minute walking distance.

### **Computing**

We have access to the OU supercomputer at the [OU Supercomputing Center for Education & Research (OSCER)](http://www.ou.edu/oscer). We obtain high-performance computing allocations on Titan from [Oak Ridge Leadership Computing Facility](https://www.olcf.ornl.gov/) (OLCF) through user proposals. Titan is a 20+ petaflops Cray XK7 supercomputer with a CPU/GPU hybrid architecture.[<img class="aligncenter wp-image-525 size-full" src="https://www.omicsbio.org/wp-content/uploads/2014/09/Titan_render.png" alt="Titan_render" width="757" height="354" srcset="https://www.omicsbio.org/wp-content/uploads/2014/09/Titan_render.png 757w, https://www.omicsbio.org/wp-content/uploads/2014/09/Titan_render-300x140.png 300w" sizes="(max-width: 757px) 100vw, 757px" />](https://www.olcf.ornl.gov/titan/)

### Mass spectrometry

We have access to [the University of Oklahoma Mass Spectrometry Facility](http://www.ou.edu/cas/chemistry/research/research-support-services/mass-spectrometry). We use LC-MS/MS to perform shotgun proteomics measurements as described in [this protocol](https://www.omicsbio.org/2015/04/15/protocol-for-shotgun-proteomics-mudpit/).

&nbsp;

### Sequencing

We obtain large-scale sequencing allocation from DOE [Joint Genome Institute](http://jgi.doe.gov/) (JGI) through user proposals. Small sequencing projects are completed using commercial sequencing services.

&nbsp;