---
id: 688
title: Strategies for metagenomic-guided whole-community proteomics of complex microbial environments.
date: 2014-11-02T23:11:08+00:00
author: omicsbio
layout: revision
guid: https://www.omicsbio.org/2014/11/02/254-autosave-v1/
permalink: /2014/11/02/254-autosave-v1/
---
Cantarel, B. L., Erickson, A. R., VerBerkmoes, N. C., Erickson, B. K., Carey, P. A., Chongle Pan, Shah, M., Mongodin, E. F., Jansson, J. K., Fraser-Liggett, C. M. & Hettich, R. L.

_PloS one_ **6**, e27173, doi:10.1371/journal.pone.0027173 (2011). [Link to Article]

<!--more-->

**Abstract:** Accurate protein identification in large-scale proteomics experiments relies upon a detailed, accurate protein catalogue, which is derived from predictions of open reading frames based on genome sequence data. Integration of mass spectrometry-based proteomics data with computational proteome predictions from environmental metagenomic sequences has been challenging because of the variable overlap between proteomic datasets and corresponding short-read nucleotide sequence data. In this study, we have benchmarked several strategies for increasing microbial peptide spectral matching in metaproteomic datasets using protein predictions generated from matched metagenomic sequences from the same human fecal samples. Additionally, we investigated the impact of mass spectrometry-based filters (high mass accuracy, delta correlation), and de novo peptide sequencing on the number and robustness of peptide-spectrum assignments in these complex datasets. In summary, we find that high mass accuracy peptide measurements searched against non-assembled reads from DNA sequencing of the same samples significantly increased identifiable proteins without sacrificing accuracy.